### DISCUSSION TITLE
[BUG] Error trying to load codespaces

### DISCUSSION THREAD

**Discussion by Nuckerr** [community/community#17971] [Codespaces] [answered]
When I try to load a codespace from a private repository I get this error.
![image](https://user-images.githubusercontent.com/62883418/172131859-6952f6f3-bbde-427d-b80d-7614f51e6f7d.png)
```
VSCodeWorkbench mount failed with "Failed to connect to relay url Code: 1006 Reason: "
```

**Comment by syedwasiraza** [comment, marked as answer]
I think U need to contact the support team....
Else another solution is that u have to delete this codespace...I'm using codespace and didn't encounter this type of error till now. This error is from server side.
Thanks.

**Reply by Nuckerr** [reply]
This happened when I created the codespace (I tried multiple times with new ones). It worked fine on my pc, this was on another machine. Where do I go to contact the team?

**Reply by syedwasiraza** [reply]
![Web capture_6-6-2022_235015_github com](https://user-images.githubusercontent.com/68981772/172227090-4b405d53-018a-4438-af8e-e7d65963ade2.jpeg)
click here and mail them...explain the problem to them and they will answer it in 2 to 3 days
